Run-LisaV2.ps1

```diff
@@ -86,6 +86,7 @@ Param(
 	# [Optional] Parameters for changing framework behavior.
 	[int]    $TestIterations = 1,
 	[string] $XMLSecretFile = "",
+	[string] $SSHPrivateKey = "",
```

Comment:
could you help me understand why SSHPrivateKey is in the parameter, and SSHPublicKey is in the secret file?
Also can we add document about how to generate them?

Reply:
SSHPublicKey - it is a string I set in XMLSecretFile file, pass it into json template when deploy VM.
SSHPrivateKey - it is a file, suffix is .ppk. also it is a string. But I didn't find a way to convert this sting to the private key file. Thoughts?

Reply:
@lizzha I update code to pass SSHPrivateKey and SSHPublicKey into the command line, let know any concern.

Reply:
Yep, I think we need to add contexts about how to use in the document.